experimental/dds/tree2/src/feature-libraries/fence.json

```diff
@@ -1,5 +1,5 @@
 {
 	"tags": ["feature-libraries"],
 	"exports": ["index"],
-	"imports": ["codec", "events", "util", "core", "shared-tree-core", "external-utilities", "codec"]
+	"imports": ["codec", "events", "util", "core", "shared-tree-core", "shared-tree"]
```

Comment:
feature libraries are consumed by shared-tree, so this is a cycle. Do not add this dep. See the diagram in the readme before making fence chance (and update it when making fence changes).

Reply:
removed shared-tree from imports